src/gd_topal.c

```diff
@@ -1800,4 +1800,34 @@ static int gdImageTrueColorToPaletteBody (gdImagePtr oim, int dither, int colors
 	return conversionSucceeded;
 }
 
+#else
```

Comment:
This `#else` appears to match `#ifdef QUANT_2PASS_SUPPORTED`, so this is not related to liq. Actually, liq is optional, see [gdPaletteQuantizationMethod](https://libgd.github.io/manuals/2.2.5/files/gd-h.html#gdPaletteQuantizationMethod). It seems to me we should rather remove the `#ifdef QUANT_2PASS_SUPPORTED`, since we have `#define QUANT_2PASS_SUPPORTED` above, so one would have to modify the file to influence it.

Reply:
thanks.  i had checked the diff files to make sure they were being compiled, and this one i couldn't get to trigger.  i just assumed i wasn't using the right configure flags to disable the features.